source/plugin/bukkit-sponge-compare.rst

```diff
@@ -0,0 +1,680 @@
+command.defaults.TeleportCommand                       (none)
+command.defaults.TellCommand                           (none)
+command.defaults.TestForCommand                        (none)
+command.defaults.TimeCommand                           (none)
+command.defaults.TimingsCommand                        (none)
+command.defaults.ToggleDownfallCommand                 (none)
+command.defaults.VanillaCommand                        (none)
+command.defaults.VersionCommand                        (none)
+command.defaults.WeatherCommand                        (none)
+command.defaults.WhitelistCommand                      (none)
+command.BlockCommandSender                             command.source.CommandBlockSource
+command.Command                                        command.CommandCallable
+command.CommandException                               command.CommandException
+command.CommandExecutor
+command.CommandMap                                     (none)
+command.CommandSender                                  command.CommandSource
+command.ConsoleCommandSender                           command.source.ConsoleSource
+command.FormattedCommandAlias                          (none)
+command.MultipleCommandAlias                           (none)
+command.PluginCommand                                  (none)
+command.PluginCommandYamlParser                        (none)
+command.PluginIdentifiableCommand                      (none)
+command.ProxiedCommandSender                           command.source.ProxySource
+command.RemoteConsoleCommandSender                     command.source.RconSource
+command.SimpleCommandMap                               (none)
+command.TabCommandExecutor                             (none)
+command.TabCompleter                                   (none)
+command.TabExecutor                                    (none)
+configuration.file.FileConfiguration                   (none)
+configuration.file.FileConfigurationOptions            (none)
+configuration.file.YamlConfiguration                   (none)
+configuration.file.YamlConfigurationOptions            (none)
+configuration.file.YamlConstructor                     (none)
+configuration.file.YamlRepresenter                     (none)
+configuration.serialization.ConfigurationSerializable  (none)
+configuration.serialization.ConfigurationSerialization (none)
+configuration.serialization.DelegateDeserialization    (none)
+configuration.serialization.SerializableAs             (none)
+configuration.Configuration                            (none)
+configuration.ConfigurationOptions                     (none)
+configuration.ConfigurationSection                     (none)
+configuration.InvalidConfigurationException            (none)
+configuration.MemoryConfiguration                      (none)
+configuration.MemoryConfigurationOptions               (none)
+configuration.MemorySection                            (none)
+conversations.BooleanPrompt
```

Comment:
For all in `conversations.`: https://github.com/SpongePowered/SpongeAPI/issues/432